lib/components/CheckBox/stories.tsx

```diff
@@ -0,0 +1,142 @@
+import { boolean, text } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/react';
+import React, { useState } from 'react';
+import { CheckBox } from '.';
+import styles from '../CheckableBase/style.scss';
+
+const baseProps = () => ({
+	label: text('Checkbox label', 'check me!'),
+	onChange: checked => console.log('checked', checked),
+});
+
+const checked = () => ({
+	checked: boolean('checked', false),
+});
+
+storiesOf('Components|CheckBox', module)
+	.addDecorator(story => (
+		<div style={{ maxWidth: '300px', width: '100%' }}>{story()}</div>
+	))
+	.add('default', () => {
+		const Example = () => {
+			const list: Array<{ label: string; value: string }> = [
+				{
+					label: 'Avocado',
+					value: 'avocado',
+				},
+				{
+					label: 'Blueberries',
+					value: 'blueberries',
+				},
+				{
+					label: 'Cherries',
+					value: 'cherries',
+				},
+				{
+					label: 'Coconut',
+					value: 'coconut',
+				},
+				{
+					label: 'Strawberries',
+					value: 'strawberries',
+				},
+			];
+
+			const [checkedValues, setCheckedValues] = useState(
+				list.reduce(
+					(values, item: { label: string; value: string }) => {
+						values[item.value] = false;
+
+						return values;
+					},
+					{}
+				)
+			);
+
+			return (
+				<>
+					{list.map((item: { label: string; value: string }) => {
+						const onClick = () =>
+							setCheckedValues({
+								...checkedValues,
+								[item.value]: !checkedValues[item.value],
+							});
+						const onChange = (checked: boolean) =>
+							console.log({ [item.value]: checked });
```

Comment:
```suggestion
							action('checked for')({ [item.value]: checked })
```

or pick something that makes sense in the action